lock_windows.go

```diff
@@ -0,0 +1,46 @@
+package dtest
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/datawire/dlib/dlog"
+	"golang.org/x/sys/windows"
+)
+
+const pattern = "C:\\Windows\\Temp\\datawire-machine-scoped-%s.lock"
```

Comment:
Tip: Use backticks instead of double quotes to avoid needing to double the backslashes
```go
const pattern = `C:\Windows\Temp\datawire-machine-scoped-%s.lock`
```